specificatie/verblijfplaats.yaml

```diff
@@ -49,11 +49,16 @@ components:
       type: string
       pattern: ^[a-zA-Z0-9 \-]{1,4}$
       example: "bis"
-    KorteNaam:
+    KorteStraatnaam:
```

Comment:
Is dit wel de goede naam? Volgens de description kan het ook de straatnaam zijn van een niet-BAG adres.

Reply:
Wellicht zijn de volgende namen een optie 
* alternatieveStraatNaam
* aliasStraatNaam
* officieuzeStraatNaam
* informeleStraatNaam

en anders inspireert het anderen misschien.

Reply:
Vanochtend besproken dat we KorteStraatnaam gebruiken. In de overgrote meerderheid van adressen is dit de -zo nodig- verkorte versie van de openbare ruimtenaam. Voor de rest is het een door de persoon opgegeven straatnaam, maar nog steeds maximaal 24 tekens lang.